packages/client/package.json

```diff
@@ -3,17 +3,20 @@
   "version": "0.3.0",
   "repository": "git@github.com:LedgerHQ/wallet-api.git",
   "license": "Apache-2.0",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/index.js",
+  "module": "lib-es/index.js",
+  "types": "lib/index.d.ts",
   "files": [
-    "/dist"
+    "/lib",
+    "/lib-es"
   ],
   "scripts": {
     "format:check": "prettier --check \"src\" \"tests\"",
     "format:fix": "prettier --write \"src\" \"tests\"",
     "lint": "eslint --cache --ext .ts \"src\" \"tests\"",
     "lint:fix": "eslint --cache --fix --ext .ts \"src\" \"tests\"",
-    "build": "tsup src/index.ts --dts --format esm,cjs,iife",
+    "dev": "tsc -p prod.tsconfig.json --watch",
+    "build": "rm -rf lib/* lib-es/* && tsc -p prod.tsconfig.json && tsc -p prod-esm.tsconfig.json",
```

Comment:
How about creating a separate `clean` script for the `rm -rf lib/* lib-es/*` and make the `build` script "depend on it" (i.e: run it before the `tsc` commands)?